spring-cloud-starter-dataflow-collector-metrics/pom.xml

```diff
@@ -2,13 +2,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
-	<artifactId>spring-cloud-starter-stream-collector-metrics</artifactId>
+	<artifactId>spring-cloud-starter-dataflow-collector-metrics</artifactId>
```

Comment:
If we envision this as just one specific instance of a "collector" abstraction, then the current name makes sense, but otherwise I'd agree that "metrics collector" should map to `metrics.collector`.